File: MissingInteger/php/TestMissingInteger.php
<?php
require 'MissingInteger.php';

class Test extends PHPUnit_Framework_TestCase {

	protected $missing_integer;

	protected function setup() {
		$this->missing_integer = new \Codility\MissingInteger;
	}

	public function testSolution() {
		$this->assertEquals(5, $this->missing_integer->solution([1,3,6,4,1,2]));
	}
}

File: MissingInteger/ruby/TestMissingInteger.rb
require_relative 'MissingInteger'
require 'minitest/autorun'

class Test < MiniTest::Test

  def setup
    @missing_integer = MissingInteger.new
  end

  def test_solution
    assert_equal 5, @missing_integer.solution([1, 3, 6, 4, 1, 2])
  end

end
